mypy/fastparse.py

```diff
@@ -356,6 +359,8 @@ def do_func_def(self, n: Union[ast3.FunctionDef, ast3.AsyncFunctionDef],
                     arg_types.insert(0, AnyType(TypeOfAny.special_form))
             except SyntaxError:
                 self.fail(TYPE_COMMENT_SYNTAX_ERROR, n.lineno, n.col_offset)
+                if n.type_comment and n.type_comment[0] != "(":
+                    self.note('Suggestion: wrap input types in parens', n.lineno, n.col_offset)
```

Comment:
Have you tested what happens if there are extra spaces after `# type:`, such as `# type:<space><space>int -> int`?

Style nits: Use "argument types" instead of "input types". Spell parens as "parentheses".

Reply:
I tested it but probably should put a test for it into the test suite.

Roger.